crates/services/gas_price_service/src/v1/tests.rs

```diff
@@ -779,13 +776,19 @@ async fn uninitialized_task__init__if_metadata_behind_l2_height_then_sync() {
         empty_block_stream(),
         gas_price_db,
         FakeDABlockCost::never_returns(),
-        onchain_db,
+        onchain_db.clone(),
     )
     .unwrap();
 
     // when
-    service.init(&StateWatcher::started()).await.unwrap();
+    let gas_price_service = service.init(&StateWatcher::started()).await.unwrap();
 
     // then
-    // no panic
+    // sleep to allow the service to sync
+    tokio::time::sleep(std::time::Duration::from_millis(100)).await;
```

Comment:
personally I'm not a fan of sleep of sleeps to wait for a condition to happen, but this is orthogonal to the PR.

Reply:
Agree, but with tokio time I can accept it since tokio auto-advances time when all futures are pending.

(Reference https://docs.rs/tokio/latest/tokio/time/fn.pause.html)